src/pal/pal_open_enclave.h

```diff
@@ -19,6 +19,9 @@ namespace snmalloc
      * PAL supports.
      */
     static constexpr uint64_t pal_features = 0;
+
+    static constexpr size_t OS_PAGE_SIZE = 0x1000;

```

Comment:
OE is somewhat interesting, because its back end doesn't actually care about pages.  I wonder what happens if we use a smaller than 4KiB page here...

Reply:
Well, experimentally dropping it to `0x800` and re-running the `two_alloc_types` suggests that things don't immediately fall over (that is, the tests pass, but I don't think that one test has phenomenal coverage of OE).